task.py

```diff
@@ -0,0 +1,13 @@
+class ToDoList:
```

Comment:
@VishnoiAman777 Hey do we have to implement the methods in here one by one?

Reply:
THis is to implement methods